services/QuillLMS/spec/workers/admin_dashboard/admin_account_created_email_worker_spec.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+# perform(user_id, admin_user_id, school_id, is_reminder)
+
+describe AdminDashboard::AdminAccountCreatedEmailWorker, type: :worker do
+  let!(:teacher) { create(:teacher) }
+  let!(:referring_admin) { create(:teacher) }
+  let!(:school) { create(:school) }
+  let!(:mailer_user) { Mailer::User.new(teacher) }
+  let!(:mailer_class)  { AdminDashboardUserMailer }
+  let!(:mailer_method) { :admin_account_created_email}
+  let!(:worker) { AdminDashboard::AdminAccountCreatedEmailWorker.new }
```

Comment:
You can get rid of `let!(:worker) { AdminDashboard::AdminAccountCreatedEmailWorker.new }` after refactoring out the `subject`